Common/Currencies.cs

```diff
@@ -305,5 +357,33 @@ public static string GetCurrencySymbol(string currency)
             string currencySymbol;
             return CurrencySymbols.TryGetValue(currency, out currencySymbol) ? currencySymbol : "$";
         }
+
+        /// <summary>
+        /// Combines all currencies pairs into a network. Used for searching for closest path.
+        /// </summary>
+        public static readonly CurrencyGraph Graph;
```

Comment:
We can move this yeah. I just need more info how/where, so for now I will postpone until it's more clear.

Further research:

ISetupHandler has function `Setup(args)`, is this where should Graph for CashBook be initialized?
Also, there would probably be needed second interface for pulling all pairs from exchange, such as ITradingPairsProvider (which could be static (just a list), or dynamic (pulls from exchange APIs) ).

Reply:
Construction of a graph is ideally self-contained in the implementation, but since we're doing a brute force, linear search, I don't think we _need_ any graph. We just need a few loops, right?

Regarding the 'pairs provider' -- since we're only dealing with static right now, let's just have the conversion provide initialize from the static currency pairs list.